src/main/java/com/umbr3114/controllers/DropController.java

```diff
@@ -328,4 +328,45 @@ public String provideOwnerId() {
 
         return dropViewModel;
     });
+
+    /**
+     * to search for drops or posts
+     */
+    public static Route search_drops_posts = ((request, response) -> {
+
+        String userInput;
+        SearchModel searchResults = new SearchModel();
+        RequestParamHelper params = new RequestParamHelper(request);
+        userInput = params.valueOf("userInput");
+
+        FindIterable<DropModel> iterable;
+        FindIterable<PostModel> iterable2;
+
```

Comment:
Not entirely required, but `iterable` and `iterable2` could be `dropIterable` and `postIterable` to make it a little clearer which iterables they belong to.